include/irrUString.h

```diff
@@ -83,17 +72,10 @@ static const irr::u16 UTF16_LO_SURROGATE = 0xDC00;
 
 namespace irr
 {
-
 	// Define our character types.
-#ifdef USTRING_CPP0X_NEWLITERALS	// C++0x
```

Comment:
other way around, `char32_t` is a C++11 type so we should use it

Reply:
@sfan5 you mean we should keep the mapping here ?